tests/test_inventory_client.py

```diff
@@ -258,6 +350,21 @@ def test_asset_identifier(cli, init_assets):
     assert asset == init_assets[2]
 
 
+def test_asset_identifier_universe(
+        cli,
+        old_universe,
+        init_old_universe_assets):
```

Comment:
Indent with 4 spaces. Move `):` to a new line.

Reply:
👍